toolkit/docs/building/building.md

```diff
@@ -71,58 +77,119 @@ Quick Start
 ## Prerequisites
 Install prerequisites [here](prerequisites.md).
 
-## Build Tools
-By default all build commands are executed from inside the `./toolkit` folder.  A set of GO Based Tools and a Toolchain must be built before building Mariner Packages and Images.
+## Configure the Environment
+By default all build commands are executed from inside the `./toolkit` folder.
 
+A few one-time steps can be run now (they will re-run automatically later if needed)
+
+Remote files are generally only available for release branches.
+```bash
+# Get the source code
+git clone https://github.com/microsoft/CBL-Mariner.git
+cd CBL-Mariner/toolkit
+
+# Checkout the desired release branch
+git checkout 1.0
```

Comment:
Thoughts on this? `git describe --tags` will return the most recent tag for the current branch.

```suggestion
git checkout 1.0
# Checkout the latest release tag (or a previous tag if desired)
git checkout $(git describe --tags)
```

Reply:
I have created a 1.0-stable tag.  I recommend using that here.  We should add a page that describes the Mariner Git Hub Tags, but for this PR...just say 1.0-stable.

---
In reply to: [489800996](https://github.com/microsoft/CBL-Mariner/pull/105#discussion_r489800996) [](ancestors = 489800996)